### DISCUSSION TITLE
Upload Flightplan via Mavlink Telemetry Failed

### DISCUSSION THREAD

**Discussion by jiratto** [svpcom/wfb-ng#286] [General]
I send and receive telemetry via mavlink it's wotk well but when uploading flightplan an timeout error occurs many times (18 times failed 2 times success).

How to config wfb_tx and wfb_rx params for upload flightplan?

**Comment by svpcom** [comment]
Do you use full wfb-ng setup on both ends or openipc on the drone side?

**Reply by jiratto** [reply]
yes. I installed 22.09 version both side.

**Reply by svpcom** [reply]
1. Check that uart is not used as serial console
2. Check telemetry with video stream (camera) disabled
3. If previous doesn't help, then try set uart speed (both on wfb and autopilot side) to 1500000 (1.5mbit)
